frontend/src/components/Snek/SnekN.js

```diff
@@ -1,20 +1,27 @@
 import React from "react";
+import PropTypes from "prop-types";
 
-const SvgSnekN = props => (
-  <div className="snektion snektion--n">
-    <svg viewBox="0 0 144.23 149.73" {...props}>
-      <style />
-      <g id="snek-n_svg__Layer_1">
-        <g id="snek-n_svg__g260_2_" transform="translate(172.282 1437.914)">
-          <path
-            id="snek-n_svg__path262_2_"
-            className="snek-body"
-            d="M-48.04-1416.06c12.89 13.51 19.99 31.74 19.99 51.34v44.19c0 8.75-3.29 16.85-9.28 22.84-6.14 6.14-14.24 9.51-22.83 9.51-8.56 0-16.91-3.42-22.95-9.39-6.28-6.09-9.87-14.44-9.87-22.95v-44.19c0-2.88-.72-5.43-1.93-6.83l-.12-.14c-.3-.36-1.1-1.3-3.83-1.3-2.52 0-4.61.82-6.38 2.5l-.16.15c-1.3 1.2-1.95 2.96-1.95 5.25v44.55c0 8.59-3.38 16.7-9.52 22.84l-.24.24c-6.23 5.98-14.34 9.28-22.83 9.28-8.46 0-16.78-3.47-22.83-9.51-6.14-6.14-9.51-14.25-9.51-22.84v-44.19c0-19.37 7.66-37.7 21.58-51.61 13.92-13.92 32.33-21.58 51.85-21.58 19.47-.01 37.54 7.77 50.81 21.84"
-          />
-        </g>
-      </g>
-    </svg>
-  </div>
-);
+const SvgSnekN = ({ up }) => {
+  const upSnek = "M 20 40 C 10 -40, 90 -40, 80 40"
+  const downSnek = "M -20 100 C 20 100, 80 100, 120 100"
+  return(
+    <div className="snektion snektion--n">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="-40 -50 180 180">
+      <style></style>
+      <path
+      stroke-width="40"
+      stroke-linecap="round"
+      stroke="#5b57a6"
+      fill="transparent"
+      class="snake-body"
```

Comment:
@mermop I noticed you changed most of the classNames from the SVG path into 'class', except for SnekEnd.js. Did you want me to change that to match also?